Readme.md

```diff
@@ -1074,6 +1074,10 @@ object. Additionally they typically come with two extra properties:
 [Error]: https://developer.mozilla.org/en/JavaScript/Reference/Global_Objects/Error
 [MySQL server error]: http://dev.mysql.com/doc/refman/5.5/en/error-messages-server.html
 
+In addition, MySQL server errors will have an `err.sql` property that contains the SQL
```

Comment:
Can you move this up to the list right above that enumerates all the properties? Seems weird for this to randomly be it's own paragraph when there is an enumeration of the properties just above.